### PR TITLE
Catch formatting errors and exit 1

### PR DESCRIPTION
**Pull request by bcomnes**
Closes https://github.com/maxogden/standard-format/issues/53

Going to think about this for a bit before closing.  Any feedback on this approach appreciated.

### CODE REVIEW

**Comment by Flet** [comment]
:+1:

**Comment by bcomnes** [comment]
Merging this as is.

File: bin.js [hunk 1]
```diff
@@ -74,8 +74,10 @@ function getFiles (done) {
 getFiles(function (err, files) {
   if (err) return error(err)
   files.forEach(function (file) {
-    file.data = fmt.transform(file.data)
-    processFile(file)
+    try {
+      file.data = fmt.transform(file.data)
+      processFile(file)
+    } catch (e) {error(file.name + ': ' + e)}
```
Comment (pdehaan) [inline bin.js]: I ran this file through `standard-format -w` and the `git diff`returned this:

```
@@ -77,7 +77,7 @@ getFiles(function (err, files) {
     try {
       file.data = fmt.transform(file.data)
       processFile(file)
-    } catch (e) {error(file.name + ': ' + e)}
+    } catch (e) {error(file.name + ': ' + e) }
   })
```

Why would it add a space after the last `)` but not move the entire `error(file.name + ': ' + e)` block onto a separate line?

**Comment by bcomnes** [comment]
I'm not exactly sure, lets turn that into another issue.

**Comment by pdehaan** [comment]
Done; filed #64

**Pull request merged**